### PR TITLE
Bump vsphere charts k8s 126

### PR DESCRIPTION
**Pull request by a-blender**
## Issue: 

vSphere CPI chart updates to upstream v1.26.0 / v1.25.1 / v1.24.4 / v1.23.4 / v1.22.8 https://github.com/rancher/rancher/issues/40908
[RFE] vSphere CSI chart updates to upstream v3.0.0 / v2.6.3 https://github.com/rancher/rancher/issues/41017
k8s 1.26 support for rancher-vsphere-csi chart https://github.com/rancher/rancher/issues/41513
k8s 1.26 support for rancher-vsphere-cpi chart https://github.com/rancher/rancher/issues/41512
[RFE] Fix vSphere CSI images installed by default https://github.com/rancher/rancher/issues/41098
Add option to enable vSphere toplogy in rancher-vsphere RKE1/RKE2 charts https://github.com/rancher/rancher/issues/40727

## Problem

Bump vsphere CPI/CSI charts and update the CSI node images that are installed by default.

## Solution

1. Bump CPI chart to `103.0.0+up1.5.1` 
2. Bump CPI chart to `103.0.0+up1.0.3-rancher1`
3. Update CPI and CSI node images that install by default to `latest` so the minimum image version doesn't get installed and it's obvious when k8s constraints aren't working properly.

### Manual Testing

Ran `make validate` to validate against upstream.

### CODE REVIEW

File: packages/rancher-vsphere/rancher-vsphere-cpi/package.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 url: https://github.com/rancher/vsphere-charts.git
 subdirectory: charts/rancher-vsphere-cpi
-commit: 5bcff9e69b932cb9c86bd1061b2ee4b5367ecb60
-version: 102.0.0
+commit: 8b8e8cf13e9c971330bf96517fab1deec1f23b05
+version: 103.0.0
```
Comment (jiaqiluo) [inline packages/rancher-vsphere/rancher-vsphere-cpi/package.yaml]: We should bump the minor version in the event of adding support for new k8s versions. 

```suggestion
version: 102.1.0
```

File: packages/rancher-vsphere/rancher-vsphere-csi/package.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 url: https://github.com/rancher/vsphere-charts.git
 subdirectory: charts/rancher-vsphere-csi
-commit: 96687762c6642efc3650626a0738c8caa45ae7c3
-version: 102.0.0
+commit: 8b8e8cf13e9c971330bf96517fab1deec1f23b05
+version: 103.0.0
```
Comment (jiaqiluo) [inline packages/rancher-vsphere/rancher-vsphere-csi/package.yaml]: We should bump the minor version in the event of adding support for new k8s versions. 
```suggestion
version: 102.1.0
```

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

**Review by jiaqiluo — APPROVED** [review]

**Review by brandond — APPROVED** [review]

**Review by brandond — APPROVED** [review]

File: packages/rancher-vsphere/rancher-vsphere-cpi/generated-changes/patch/Chart.yaml.patch [hunk 3]
```diff
@@ -8,8 +8,8 @@
 +  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
    catalog.cattle.io/release-name: vsphere-cpi
  apiVersion: v1
--appVersion: 1.4.1
-+appVersion: 1.4.2
+-appVersion: 1.26.1
++appVersion: 1.5.1
```
Comment (snasovich) [inline packages/rancher-vsphere/rancher-vsphere-cpi/generated-changes/patch/Chart.yaml.patch:12]: Just curious, why we need to override this? The "app" is vSphere CPI chart so it makes sense to track the (latest) version of vSphere CPI plugin included.
This was done in `vsphere-charts` superchart recently - https://github.com/rancher/vsphere-charts/pull/55/files#diff-50ddca5e4037a6988a4f9eb12c30c2c5fa8730fb2cc7d07f1517feee49e01804R11

Comment (a-blender) [reply packages/rancher-vsphere/rancher-vsphere-cpi/generated-changes/patch/Chart.yaml.patch:12]: I was following the pattern of the previous charts https://github.com/rancher/charts/blob/37a92a5927ad61a29bb42558d4ade53a9595a43f/packages/rancher-vsphere/rancher-vsphere-cpi/generated-changes/patch/Chart.yaml.patch#L11-L12 and we override the `appVersion` to match the superchart version.

Comment (snasovich) [reply packages/rancher-vsphere/rancher-vsphere-cpi/generated-changes/patch/Chart.yaml.patch:12]: @a-blender , got it! Thank you.

**Pull request merged**